Tests/Tests/HttpRequestableSpec.swift

```diff
@@ -320,6 +323,41 @@ class HttpRequestableSpec: QuickSpec {
                         }
                     }
                 }
+
+                it("will using empty string for error message as a default when failed to encode error message as .utf8") {
+                    let data = "テスト".data(using: .japaneseEUC)!
+                    let response = HTTPURLResponse(
+                        url: URL(string: "https://test.url")!,
+                        statusCode: 404,
+                        httpVersion: nil,
+                        headerFields: nil)
+
+                    httpRequestable.httpSessionMock.responseData = data
+                    httpRequestable.httpSessionMock.httpResponse = response
+
+                    waitUntil { done in
+                        requestQueue.async {
+                            let result = httpRequestable.requestFromServerSync(
+                                url: URL(string: "https://test.url")!,
+                                httpMethod: .put,
+                                parameters: nil,
+                                addtionalHeaders: nil)
+
+                            expect {
+                                try result.get()
+                            }.to(throwError())
+                            done()
+                        }
+                    }
+                }
+
+                it("will throw assertion if calling the method on the main thread") {
+                    expect(httpRequestable.requestFromServerSync(
```

Comment:
Please add a comment here that `throwAssertion()` captures a strong reference to `httpRequestable` and never releases it making it impossible to deallocate the `httpSessionMock` reusable instance

Reply:
Added here [be853e2](https://github.com/rakutentech/ios-inappmessaging/pull/273/commits/be853e2b2b0b107e942a87a69858c3b40da557ec).

Reply:
Thanks